examples/grids/HV_9_Bus_Set1_steadystate_case.json

```diff
@@ -232,11 +247,7 @@
       "name": "StatGenerator_3",
       "reactive_power": {
         "controller": {
-          "control_type": {
-            "control_strategy": "COSPHI_CONST",
-            "cosphi": 0.7,
-            "cosphi_dir": "UE"
-          },
+          "control_type": {},
```

Comment:
?

Reply:
this is happen because of change in `controller `in psdm PR [pydanctic v2](https://github.com/ieeh-tu-dresden/power-system-data-model/pull/36)
we have to fix psdm first